.travis.yml

```diff
@@ -1,49 +1,40 @@
-language: cpp
+language: generic
+sudo: required
 dist: trusty
-sudo: false
+
 matrix:
-    include:
-        - os: linux
-          compiler: clang
-          env: BUILD_TYPE=Dev
-        - os: linux
-          compiler: clang
-          env: BUILD_TYPE=Release
-        - os: linux
-          compiler: gcc
-          env: BUILD_TYPE=Dev
-        - os: linux
-          compiler: gcc
-          env: BUILD_TYPE=Release
-        - os: osx
-          compiler: clang
-          env: BUILD_TYPE=Dev
-        - os: osx
-          compiler: clang
-          env: BUILD_TYPE=Release
+  fast_finish: true
+  include:
+    - os: linux
+      addons:
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['g++-7', 'cmake', 'realpath', 'libbz2-dev', 'zlib1g-dev', 'libexpat1-dev', 'libboost-all-dev']
```

Comment:
Thanks @jenningsanderson for fixing in https://github.com/mapbox/osm-tag-history/commit/36adfbe27cf477d828a6fe9fb4599d1dc9b09f20